pkg/api/v1beta3/types.go

```diff
@@ -1691,7 +1691,7 @@ type ComponentCondition struct {
 // ComponentStatus (and ComponentStatusList) holds the cluster validation info.
 type ComponentStatus struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"standard object metadata; see https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#metadata"`
 
 	Name       string               `json:"name,omitempty" description:"name of the component"`
 	Conditions []ComponentCondition `json:"conditions,omitempty" description:"Conditions is a list of ComponentCondition objects"`
```

Comment:
The patch-related tags are mainly intended for writes, but I'd like all maps represented as lists in the API to be tagged, in part because we're moving in the direction of all status being POSTed/PUT to the apiserver.